src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
The function name `makeCalculator` is appropriate as per the naming conventions in the checklist.